src/legacy/server/kbn_server.js

```diff
@@ -45,9 +45,21 @@ import { uiMixin } from '../ui';
 import { sassMixin } from './sass';
 import { i18nMixin } from './i18n';
 
+/**
+ * @typedef {import('./kbn_server').KibanaConfig} KibanaConfig
```

Comment:
Added a few of these comment `typedef`s to improve IDE intellisense when working in these files.